apps/passport-server/src/services/generic-issuance/genericIssuanceService.ts

```diff
@@ -366,7 +422,17 @@ export class GenericIssuanceService {
     userId: string
   ): Promise<PipelineDefinition[]> {
     // TODO: Add logic for isAdmin = true
-    return (await this.definitionDB.loadPipelineDefinitions()).filter((d) =>
+    const allDefinitions: PipelineDefinition[] =
+      await this.definitionDB.loadPipelineDefinitions();
+    let a: number = "5";
+
+    const allDefinitions2: PipelineDefinition[] =
+      await this.definitionDB.loadPipelineDefinitions();
+    const allDefinitions3: PipelineDefinition[] =
+      await this.definitionDB.loadPipelineDefinitions();
+    const allDefinitions4: PipelineDefinition[] =
+      await this.definitionDB.loadPipelineDefinitions();
```

Comment:
I'm not totally sure what is going on here?

Reply:
was trying to figure out what [was going on here](https://discord.com/channels/1096493555033325578/1138110482595708998/1203149859784499260) - removed for the real version of this PR